templates/document.html.haml

```diff
@@ -50,7 +51,6 @@
       - if attr? key
         %meta{:name=>(meta || key), :content=>(attr key)}
 
-    %title=((doctitle :sanitize => true) || (attr 'untitled-label'))
```

Comment:
Did you mean to remove the `<title>` tag?

Reply:
there's a title tag getting added by cp-chrome